ansible/playbooks/sap-hana-install.yaml

```diff
@@ -5,7 +5,7 @@
   become_user : root
   pre_tasks:
     - name: Source hana install variables
-      include_vars: ./vars/sap-hana-install_vars.yaml
+      include_vars: ./vars/hana-vars.yaml
```

Comment:
How can I tell which of the variables from hana/vars are needed in this playbook? Seems **sap_hana_install_sid**
 is one of them. Can we think about something to help to identify them?